views/github-create-branch.hbs

```diff
@@ -63,7 +63,7 @@
     <div class="field-group">
       <aui-label for="branch-name">Branch name</aui-label>
       <input class="text full-width-field" type="text" data-aui-validation-field required name="branch-name"
-             placeholder="Enter branch name"/>
+             value="{{issue.key}}" placeholder="Enter branch name"/>
```

Comment:
Add the issue summary in the default name... if it exists.
kebab cased and all lower case to incase it comes through strange.
e.g. `key-99-summary-of-goodies`

strip out all none alphanumeric characters toooooooo

Reply:
Combined summary and key for the branch name.
Added a new helper for removing non alphanumeric chars.